run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+poetry install
```

Comment:
Your production image already includes `RUN poetry install`.

So really this file is just the `poetry run gunicorn ...` line, so you don't really need a separate file for it.